Makefile

```diff
@@ -90,7 +90,7 @@ build-iso:
 	mkdir -p $(ROOT_DIR)/build
 	$(DOCKER) run --rm -v $(DOCKER_SOCK):$(DOCKER_SOCK) -v $(ROOT_DIR)/build:/build \
 		--entrypoint /usr/bin/elemental $(TOOLKIT_REPO):$(VERSION) --debug build-iso --bootloader-in-rootfs -n elemental-$(FLAVOR).$(ARCH) \
-		--local --platform $(PLATFORM) --squash-no-compression -o /build $(REPO):$(VERSION)
+		--local --platform $(PLATFORM) -o /build $(REPO):$(VERSION)
```

Comment:
Will it default to compressed squashfs images?

Reply:
Only for our example isos/disks since I noticed we saved ~50MB on the final image which could speed up CI sligthly.